instructions/fault-injection/fault-injection.adoc

```diff
@@ -0,0 +1,150 @@
+= Chaos Testing
+include::_attributes.adoc[]
+
+Apply some chaos engineering by throwing in some HTTP errors or network delays. Understanding failure scenarios is a critical aspect of microservices architecture (aka distributed computing)
+
+[IMPORTANT]
+.Before Start
+====
+You should have only the following virtualservices and destinationrules in the `istio-tutorial` namespace
+
+[source, bash,subs="+macros,+attributes"]
+----
+$ oc -n istio-tutorial get destinationrule
+No resources found.
+
+$ oc -n istio-tutorial get virtualservice
```

Comment:
We need to be consistent with the namespace, the kiali lab uses tutorial and not istio-tutorial.  Should we update this to istio-lab or something similar?

Reply:
We can always change this but Erik's latest email said he's using istio-tutorial for the lab environement so I'm matching that.

Reply:
Ah okay, I'll go and update all my PRs in that case since they all refer to 'tutorial' rather than 'istio-tutorial'